### ISSUE TITLE
Unable to configure field display with Tripal DS enabled

### ISSUE THREAD

**Issue by laceysanderson** [tripal/tripal#826]
### System information

* Tripal Version: 7.x-3.x dev (3244f2dc184513d1c4576fb29b2d72e6f61d4413)
* Drupal Version: 7.63
* PostgreSQL Version: 9.4.10
* PHP Version: 7.1.25

### Issue description
When configuring the display for any Tripal content type, I am unable to configure any field. For example, when trying to configure the relationships formatter for an organism, I click on the cog next to the field on Admin > Structure > Tripal Content Types > Organism > Manage Display and the ajax triggers but no blue configuration box appears. 

There are no errors on the page: the admin simply sees the ajax spinner and then the page reverts back to exactly what it was before the cog was clicked. 

In the Drupal Watchdog logs, you see the following two errors:

 - `Invalid form POST data`
 - `Notice: Undefined index: form_build_id in ajax_get_form() (line 325 of /home/www/tripal/includes/ajax.inc).`

##### With Tripal DS **Enabled**
1. Navigate to Admin > Structure > Tripal Content Types > [Content Type of your choice] > Manage Display
2. Click on "Apply Default Tripal Layout" and confirm
3. Click on the cog/gear icon to the right of any field that supports configuration
You will see an ajax spinner saying "please wait" appear next to the cog/gear and then it will disappear without modifying the page.
<img width="1387" alt="screen shot 2019-01-22 at 9 07 34 am" src="https://user-images.githubusercontent.com/1566301/51544446-94c3e480-1e25-11e9-918a-35f9b3d2ab67.png">

##### How to fully disable Tripal DS to see orginal functionality
1. Disable both Tripal DS and Display Suite
2. Navigate to the content type you will be testing and remove all Tripal Panes.

##### With Tripal DS **Disabled**
1. Navigate to Admin > Structure > Tripal Content Types > [Content Type of your choice] > Manage Display
2. Click on the cog/gear icon to the right of any field that supports configuration.
You will see an ajax spinner saying "please wait" appear next to the cog/gear and then a blue section will appear with configuration for that field as shown in the following screenshot.
<img width="1390" alt="screen shot 2019-01-22 at 8 59 14 am" src="https://user-images.githubusercontent.com/1566301/51544058-a5c02600-1e24-11e9-86f4-9d5f1bc4a2e0.png">

**Comment by laceysanderson** [comment]
I'm at a bit of a loss here. @shawnawsu  do you have any ideas what might be causing this?

**Comment by shawnawsu** [comment]
@laceysanderson  I cannot for the life of me reproduce this. I've tried different php and drupal version combos and different branches. Are you still experiencing this?

**Comment by laceysanderson** [comment]
🎉 I was experiencing this still about 5 days ago but just checked again and it's working!!! Unfortunately I have no idea what has changed so I guess we'll just hope no one runs across it again? 🤷‍♀️ 

The weird thing is when I first started experiencing it, it was very reproducible. I was able to re-create it on both KnowPulse clones and generic Tripal sites repeatedly.

Thank you Very much for looking into this @shawnawsu
